packages/flutter/lib/src/widgets/checked_mode_banner.dart

```diff
@@ -66,12 +111,34 @@ class CheckedModeBanner extends StatelessWidget {
   Widget build(BuildContext context) {
     Widget result = child;
     assert(() {
-      result = new CustomPaint(
-        foregroundPainter: const _CheckedModeBannerPainter(),
-        child: result
-      );
+      result = new Banner(
+        child: child,
```

Comment:
If we later add another thing around this (like, say, ExcludeSemantics, which come to think maybe we should) then it won't be randomly buggy in one of the code paths.